lib/js/jsconsole.nim

```diff
@@ -0,0 +1,32 @@
+#
+#
+#            Nim's Runtime Library
+#        (c) Copyright 2012 Andreas Rumpf
+#
+#    See the file "copying.txt", included in this
+#    distribution, for details about the copyright.
+#
+
+## Wrapper for the `console` object for the `JavaScript backend
+## <backends.html#the-javascript-target>`_.
+
+when not defined(js) and not defined(Nimdoc):
+  {.error: "This module only works on the JavaScript platform".}
+
+type Console* {.importc.} = ref object of RootObj
+
+{.push importcpp .}
+
+proc log*[A](console: Console, a: A)
```

Comment:
The problem with the `logging` module is, again, the conversion to string. It is simply too inconvenient to inspect javascript object just by dumping strings in the console.

Actually, Nim objects often have a pretty simple and inspectable layout, and more so if you are just dealing with js objects, say coming from external libraries, wrapped in Nim.

Your solution almost works, but wraps everything into an array. Not a big deal for me, just a little strange. I had to change `#.log.apply(null, #)` into `#.log(#)` though, otherwise it would not work on Firefox.

If you are ok, I would just update the PR to your version